grid/app/main/sfl/models/ai_model.py

```diff
@@ -1,4 +1,4 @@
-from .. import db
+from ... import db
```

Comment:
The grouping "standard python, external module, local" is not being acknowledge here.